docker/build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.19-bullseye
+FROM golang:1.19.3-bullseye
```

Comment:
In principle images should be idempotent, but that is not the case even when 1.19.3 is explicitly specified.

The most recent (probably patch) versions of various packages are installed when the image is built. There's an `apt-get update` and `install` below in line 5.

This is no different than jumping from Go 1.19.2 to 1.19.3